letsencrypt/tests/auth_handler_test.py

```diff
@@ -479,8 +479,11 @@ def test_different_errors_and_domains(self, mock_zope):
 
 def gen_auth_resp(chall_list):
     """Generate a dummy authorization response."""
-    return ["%s%s" % (chall.__class__.__name__, chall.domain)
+    from acme.challenges import HTTP01Response
+    return [HTTP01Response(key_authorization=u'foo')
```

Comment:
Why's that? The returned HTTP01Response object is still not the actual response. I'm not exactly sure how does something classify as "dummy" when it comes to uniy testing.